src/aap_eda/settings/default.py

```diff
@@ -403,14 +414,25 @@ def _rq_redis_client_additional_parameters():
     return params
 
 
-def rq_redis_client_instantiation_parameters():
+def rq_standalone_redis_client_instantiation_parameters():
     params = _rq_common_parameters() | _rq_redis_client_additional_parameters()
 
     # Convert to lowercase for use in instantiating a redis client.
     params = {k.lower(): v for (k, v) in params.items()}
     return params
 
 
+def rq_redis_client_instantiation_parameters():
+    params = rq_standalone_redis_client_instantiation_parameters()
+
+    # Include the HA cluster parameters.
+    if REDIS_HA_CLUSTER_HOSTS:
+        params["mode"] = "cluster"
```

Comment:
@jshimkus-rh Do we need a log line to indicate that we are in Cluster mode might help in debugging if we only have the logs available.

Reply:
@mkanoor I was originally going to add one, but we aren't currently logging anything as part of default.py so I didn't.  If folks want it and are okay with it I can add it.